drivers/dai/intel/ssp/ssp.c

```diff
@@ -9,6 +9,7 @@
 #include <stdint.h>
 #include <zephyr/spinlock.h>
 #include <zephyr/devicetree.h>
+#include <zephyr/pm/device.h>
```

Comment:
Hmm, this seems to break the existing ssp driver test. I think you need to add dependency to CONFIG_PM as PM calls are used unconditionally in this driver now (and cannot compiled without CONFIG_PM).

E.g. "west build -b intel_adsp_cavs25 tests/boards/intel_adsp/ssp/" fails. Not 100% sure what is the most recommend way but least adding a "depends on PM_DEVICE_RUNTIME" to Kconfig.ssp helped to fix the build.

The test case still fails even if compiles to:

```
--
I: cavs_gpdma_init: dma DMA_1 initialized
I: cavs_gpdma_init: dma DMA_0 initialized

    Assertion failed at ZEPHYR_BASE/tests/boards/intel_adsp/ssp/src/main.c:401: test_main: (dev_dai_ssp is NULL)
```

We don't need to support runtime-pm for cAVS2.5, but we can't break existing functionality, so some solution to this is needed.